pkg/scheduler/apis/config/types.go

```diff
@@ -131,3 +143,72 @@ type KubeSchedulerLeaderElectionConfiguration struct {
 	// LockObjectName defines the lock object name
 	LockObjectName string
 }
+
+// Plugins include enabled plugins and disabled plugins, when enabled, the list of plugins for
+// a particular extension point are the only ones enabled. If an extension point is
+// omitted from the config, then the default set of plugins is used for that extension point.
+type Plugins struct {
+	// EnabledPlugins specifies plugins that should be enabled in addition to default plugins.
```

Comment:
> in addition to default plugins

if the order in these enabled slices is significant, but there can also be default plugins, in what order are the default plugins and the ordered plugins in this list combined?

Reply:
Great point. We should probably go with "defaults first" then "enabled ones". If that is not desired, one can disable defaults and re-enable everything (defaults and custom ones) in a different order.